packages/instant-meilisearch/vite.config.ts

```diff
@@ -0,0 +1,47 @@
+import { defineProject } from 'vitest/config'
+import pkg from './package.json' with { type: 'json' }
+
+const globalVarName = pkg.name
+  .substring(pkg.name.lastIndexOf('/') + 1)
+  .replace(/-./g, (x) => x[1].toUpperCase())
+
+export default defineProject({
+  build: {
+    sourcemap: true,
+    target: 'es6',
+    lib: {
+      entry: 'src/index.ts',
+      name: globalVarName,
+      formats: ['umd'],
+      fileName: (format, entryName) => {
+        switch (format) {
+          case 'umd':
+            return `umd/${entryName}.min.js`
+          default:
+            throw new Error(`unsupported format ${format}`)
+        }
+      },
+    },
+    rollupOptions: {
+      external: ['meilisearch'],
```

Comment:
I updated this part to keep `meilisearch` as an external dependency, like it was the case before.